.pre-commit-config.yaml

```diff
@@ -93,6 +93,7 @@ repos:
             --,
             --deny=warnings,
             --deny=clippy::self_named_module_files,
+            --deny=clippy::wildcard_imports
```

Comment:
so this doesn't prevent use from `pub use foo::*` reexport ?

Reply:
`clippy` don't ask me to change these import, so I would say no it allow us the use of